yew-router/examples/README.md

```diff
@@ -5,7 +5,14 @@
 - switch - Various examples for how to construct routes with the router.
 
 ## Running
-Details on how to build and run these examples can be found in the readme under `servers/`.
+
+There are two ways to run, with Rust servers or with Python.
```

Comment:
```suggestion
To run the examples you'll need to spin up a web server; two possible ways of doing this are either using a Rust webserver or the built-in Python webserver (we suggest this because Python is installed by default on most systems).
```